VideoWeb/VideoWeb/ClientApp/src/app/shared/self-test/self-test.component.ts

```diff
@@ -242,10 +242,12 @@ export class SelfTestComponent implements OnInit, OnDestroy {
             this.videoCallService.updateMicrophoneForCall(mic);
         }
         this.preferredMicrophoneStream = await this.userMediaStreamService.getStreamForMic(mic);
-        this.logger.info(`${this.loggerPrefix} Update camera and microphone selection`, {
-            cameraId: cam.deviceId,
-            microphoneId: mic.deviceId
-        });
+        if (cam && mic) {
+            this.logger.info(`${this.loggerPrefix} Update camera and microphone selection`, {
+                cameraId: cam.deviceId,
+                microphoneId: mic.deviceId
```

Comment:
if the devices are null, use the existential operator to handle null values

Reply:
I see, it was error on console, just tried to fix.   I removed 'If (cam && mic).